Development environment setup/Useful software.md

```diff
@@ -20,4 +20,7 @@ In the following list, we'll give some useful software and links that will speed
 
 [Hover inspector for Chrome](https://chrome.google.com/webstore/detail/hover-inspector-like-in-z/jbhnejndccdjigppalbbiphlbaaehpln/related?hl=en) - Inspector is the tool which mimics display dimensions between selected and hovered element. It can be useful for designers and QA engineers
 
-[Accessibility checker for Firefox](https://addons.mozilla.org/en-US/firefox/addon/accessibility-checker/) - IBM Equal Access Accessibility Checker
\ No newline at end of file
+[Accessibility checker for Firefox](https://addons.mozilla.org/en-US/firefox/addon/accessibility-checker/) - IBM Equal Access Accessibility Checker
+
+[Laravel Herd](https://herd.laravel.com/) - As it provides a simple way to set up a local development environment. Installation on a clean setup is really easy you just download the installer and run it. It will install all the necessary software and set up the environment for you.

```

Comment:
```suggestion
[Laravel Herd](https://herd.laravel.com/) - Provides a simple way to set up a local development environment. Installation on a clean setup is really easy you just download the installer and run it. It will install all the necessary software and set up the environment for you.

```